src/configs/shlvl.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct ShLvlConfig<'a> {
+    pub symbol: Option<SegmentConfig<'a>>,
+    pub threshold: i64,
+    pub prefix: &'a str,
+    pub suffix: &'a str,
+    pub style: Style,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for ShLvlConfig<'a> {
+    fn new() -> Self {
+        ShLvlConfig {
+            symbol: Some(SegmentConfig::new("↕️ ")),
+            threshold: 3,
```

Comment:
I'm curious why `3` is your default? Would `2` not make more sense? Is there some kind of common scenario you are accounting for here?

Reply:
the original issue said 3, but i can change that since it is a config option :)

Reply:
Ah, I should have read the original issue 😅 

The `tmux`/`screen` scenario makes sense.